protos/dcs/mission/v0/mission.proto

```diff
@@ -303,6 +303,21 @@ message StreamEventsResponse {
     dcs.common.v0.Airbase place = 3;
   }
 
+  // Occurs when a chat message is sent on the server
+  message PlayerSendChatEvent {
+    // The player's id in the current server session.
+    uint32 playerId = 1;
+    // what was typed
+    string message = 2;
+    // to whom?
+    MessageTarget target = 3;
+    enum MessageTarget {
+      MESSAGE_TARGET_UNSPECIFIED = 0;
+      MESSAGE_TARGET_COALITION = 1;
+      MESSAGE_TARGET_ALL = 2;
+    }
```

Comment:
Nit: Just noticed, while seeing the removed code form the previous chat stream that the enum could also be covered by

```
  // Whether the message was send to all or only to allies.
   bool all = 4;
```

But I don't have a strong opinion here, I am fine with either one.

Reply:
It also occurs to me.

With the current setup the client will now know what coalition the message is sent to based on this message. They would need to go and get the player information and infer from there.

Should we instead enumerate the coalitions instead and have the server determine what the value should be based on the player? 

If we do we could reuse the coalition enum from `common`. https://github.com/DCS-gRPC/rust-server/blob/main/protos/dcs/common/v0/common.proto#L43

Reply:
I like that idea @rurounijones. I would implement it in a heartbeat except that the `all` flag from DCS API is bugged. See my note here: https://github.com/DCS-gRPC/rust-server/pull/89#discussion_r776235960

Perhaps at this moment, knowing we have a bug which would not activate the cool API, should we just skip this field? I'll leave a comment with further extension when the DCS environment behaves correctly.

Reply:
That is a good point. I think skipping the field entirely is probably the better option as it means no backwards compatibility issues if ED every fix it and we choose to add it since there would be no existing field that needs modification.

Reply:
+1 for skipping it for now